source/delphi_gui/pasdoc_gui_Tokyo.dproj

```diff
@@ -1,206 +0,0 @@
-﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
Same comment as before: do not remove Delphi Tokyo specific files.

Reply:
Same reason here.